examples/README.md

```diff
@@ -302,8 +302,8 @@ $ ./bin/example_rng_uniform_usm
 # Using single precision (float) data type
 #
 # Device will be selected during runtime.
-# The environment variable SYCL_DEVICE_FILTER can be used to specify
-# SYCL device
+# The environment variable ONEAPI_DEVICE_SELECTOR can be used to specify
+# ONEAPI device
```

Comment:
One last place:
```suggestion
# available devices
```

Reply:
Thanks for the catch, made the change. Resolving